src/Templates/src/templates/maui-mobile/Pages/Controls/TaskView.xaml

```diff
@@ -41,10 +41,19 @@
                             Command="{Binding NavigateToTaskCommand, Source={RelativeSource AncestorType={x:Type pageModels:IProjectTaskPageModel}}, x:DataType=pageModels:IProjectTaskPageModel}" 
                             CommandParameter="{Binding .}"/>
                     </Grid.GestureRecognizers>
-                    <CheckBox Grid.Column="0" IsChecked="{Binding IsCompleted, Mode=OneTime}" VerticalOptions="Center" CheckedChanged="CheckBox_CheckedChanged"/>
-                    <Label Grid.Column="1" Text="{Binding Title}" VerticalOptions="Center" LineBreakMode="TailTruncation"/>
+                    
+                    <CheckBox Grid.Column="0"
+                        IsChecked="{Binding IsCompleted, Mode=OneTime}"
+                        VerticalOptions="Center"
+                        CheckedChanged="CheckBox_CheckedChanged"
+                        SemanticProperties.Description="{Binding Title, StringFormat='{0}'}"/>
+
+                    <Label Grid.Column="1"
```

Comment:
I think if we're adding the Title to the Checkbox then we'd set this Label to be ignored by screen readers.

Reply:
Done!